bin/cylc-scan

```diff
@@ -221,13 +221,17 @@ def main():
     if options.all_ports:
         args.extend(glbl_cfg().get(["suite host scanning", "hosts"]))
     if not args:
+        print 1, args
```

Comment:
Did you mean to add this?

Reply:
ah, bollocks, no.  Some debug left-over from the cylc scan issues today.  The dangers of rushing...